lib/icon/module.js

```diff
@@ -19,6 +19,18 @@ async function run (pwa, _emitAssets) {
   const defaults = {
     iconFileName: 'icon.png',
     sizes: [64, 120, 144, 152, 192, 384, 512],
+    iosSizes: [
+      [1536, 2048, 'ipad'], // Ipad
+      [1536, 2048, 'ipadpro9'], // Ipad Pro 9.7"
+      [1668, 2224, 'ipadpro10'], // Ipad Pro 10.5"
+      [2048, 2732, 'ipadpro12'], // Ipad Pro 12.9"
+      [640, 1136, 'iphonese'], // Iphone SE
+      [50, 1334, 'iphone6'], // Iphone 6
```

Comment:
Is the iphone 6 splash screen suppose to be 50 px wide?

Reply:
definitely is 750 x 1334